pkg/ddevapp/config.go

```diff
@@ -195,25 +195,27 @@ func (app *DdevApp) WriteConfig() error {
 // back to defaults for config values not defined in the read config file.
 func (app *DdevApp) ReadConfig() error {
 
-	source, err := ioutil.ReadFile(app.ConfigPath)
+	// Load config.yaml
+	err := app.LoadConfigYamlFile(app.ConfigPath)
 	if err != nil {
-		return fmt.Errorf("could not find an active ddev configuration at %s have you run 'ddev config'? %v", app.ConfigPath, err)
-	}
-
-	// validate extend command keys
-	err = validateCommandYaml(source)
-	if err != nil {
-		return fmt.Errorf("invalid configuration in %s: %v", app.ConfigPath, err)
+		return fmt.Errorf("unable to load config file %s: %v", app.ConfigPath, err)
 	}
-
 	app.DBImage = "" // DBImage will be set below
 
-	// ReadConfig config values from file.
-	err = yaml.Unmarshal(source, app)
+	// Load config.*.y*ml after in glob order
+	glob := filepath.Join(filepath.Dir(app.ConfigPath), "config.*.y*ml")
```

Comment:
A `config.*.y?ml` makes more sense to me.

Reply:
Thanks for taking a look @unn - but y?ml doesn't match config.*.yml, which is the intent here. It seems [glob has no concept of 0 or 1 characters](https://en.wikipedia.org/wiki/Glob_(programming)) like some versions of regex do. Of course we could use regex here.

Reply:
I tried! On the upside we can say we support `yaaaaaaaml` files.

Reply:
And config.yWHATDOYOUMEANml!

Reply:
Oh, I guess we do config.what.in.the.hell.is.in.this.yANIml!